Gemfile

```diff
@@ -37,7 +37,7 @@ gem 'jbuilder', '~> 2.0'
 # jquery multiselect plugin for advanced search
 gem 'chosen-rails'
 
-gem 'blacklight-marc', git: 'https://github.com/projectblacklight/blacklight-marc.git', ref: 'c0ff1d9'
```

Comment:
This dependency at this particular commit relies on Rails behavior that was removed in rails 6.1